### DISCUSSION TITLE
ERROR INSTALLING VS CODE EXTENSION!

### DISCUSSION THREAD

**Discussion by CodeAyoola** [microsoft/vscode-discussions#625] [Extension Development QnA] [answered]
I attempted to install extensions for VS Code, but the installation was unsuccessful.
How do I fix this ?

**Comment by alefragnani** [comment, marked as answer]
Hi @CodeAyoola ,

This may happen for a wide set of reasons, but it will depend on a more detailed analysis and details you could provide, like logs/messages. 

But, the best location for you to look for solutions of similar issues or even report your new scenario would be the VS Code repository instead (https://github.com/microsoft/vscode/issues/). This Discussions repo is focused for extension development for now.

Hope this helps

**Comment by CodeAyoola** [comment]
So sorry for the late response, I have been able to resolve it.
